scripts/lldbinit.py

```diff
@@ -44,13 +44,14 @@ def print_usage() -> None:
 Dumps the value of an associated ID, using the C++ Dump() functions.
 
 Usage:
-  dump <CONTEXT> [<EXPR>|-- <EXPR>|<TYPE><ID>|<TYPE> <ID>]
+  dump <CONTEXT> [<EXPR>|-- <EXPR>|<TYPE><ID>|<IR>.<TYPE><ID>]
```

Comment:
Minor suggestion to avoid copy-paste

```suggestion
  dump <CONTEXT> [<EXPR>|-- <EXPR>|[<IR>.]<TYPE><ID>]
```

Or be very precise about supported syntax:

```suggestion
  dump <CONTEXT> [<EXPR>|-- <EXPR>|<TYPE><ID>|ir<ID>.inst<ID>]
```

Reply:
Done. Not using the `inst` version. David has said he is planning to expand the tagging to other IDs than InstId in the future, it is just the first one.